### PR TITLE
Add support of Symfony 4

### PR DESCRIPTION
**Pull request by maidmaid**

### CODE REVIEW

**Comment by Paulmolin** [comment]
Hello @maidmaid !

Thank you for your Pull Request. I just reactivated Travis CI, can you push on your Pull Request to make sure the build is triggered?

**Comment by maidmaid** [comment]
Hello @Paulmolin, I just did it :)

File: composer.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
         }
     },
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=7.1.3",
```
Comment (benja-M-1) [inline composer.json]: We still can support php >=5.3.3 for now. 
Can you add a OR in the version please?
```suggestion
        "php": ">=5.3.3|>=7.1.3",
```

**Review by benja-M-1 — CHANGES_REQUESTED** [review]

**Comment by benja-M-1** [comment]
Hello @maidmaid :) Do you know when you will be able to update your PR?  That would be awesome to merge your PR in the week!

**Comment by maidmaid** [comment]
Hi @benja-M-1, Ok, I'm gonna do it while this week :)

**Comment by maidmaid** [comment]
It works. I think this PR could be ready as a first step and later, we can improve it in a second PR by making a code without deprecation notices from Symfony 4.

**Review by benja-M-1 — APPROVED** [review]

**Comment by benja-M-1** [comment]
Hi @maidmaid thank you very much for your help to update this bundle! I do agree, to remove deprecation notices from Symfony 4.

**Pull request merged**